From ccd5f0e93b318185b57accd7aba9764cdb13fc66 Mon Sep 17 00:00:00 2001
From: Ahmed Mansour <Ahmed.Mansour@freescale.com>
Date: Wed, 20 Jan 2016 17:22:04 -0500
Subject: [PATCH 0934/1429] dce/dpdcei-cmd: Update dpdcei minor version

Update the dpdcei version supported by the dce driver from 1.1
to 1.2

Signed-off-by: Ahmed Mansour <Ahmed.Mansour@freescale.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-dpaa2/dce/dpdcei-cmd.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/dce/dpdcei-cmd.h b/drivers/staging/fsl-dpaa2/dce/dpdcei-cmd.h
index 828c8dc..c646ad8 100644
--- a/drivers/staging/fsl-dpaa2/dce/dpdcei-cmd.h
+++ b/drivers/staging/fsl-dpaa2/dce/dpdcei-cmd.h
@@ -34,7 +34,7 @@
 
 /* DPDCEI Version */
 #define DPDCEI_VER_MAJOR				1
-#define DPDCEI_VER_MINOR				1
+#define DPDCEI_VER_MINOR				2
 
 /* Command IDs */
 #define DPDCEI_CMDID_CLOSE				0x800
-- 
1.7.5.4

